pkg/kube/descheduler-job.yaml

```diff
@@ -0,0 +1,54 @@
+---
```

Comment:
Do you know if we can license it to us? If not, we should add it to `.spdxignore`:
https://github.com/lf-edge/eve/blob/e0b943fd55e773f2000c8e63611f242f615c902c/.spdxignore

Reply:
The project is Apache 2.0, I added it to the .spdxignore for now.